playlist.c

```diff
@@ -1363,9 +1363,9 @@ static bool playlist_read_file(
 
       filestream_seek(file, 0, SEEK_SET);
 
-      if (bytes_read == 15)
+      if (bytes_read == 1)
```

Comment:
This will basically never be true because it still tries to read 15 bytes in, so a regular playlist will never be detected properly as bytes_read should always be 15.

But regardless, we need to actually conform to the spec, see my general comments in the PR.

Reply:
:+1: